src/plugins/mapset.ts

```diff
@@ -17,6 +17,7 @@ import {
 	ProxyTypeSet,
 	die
 } from "../internal"
+import {each} from "../utils/common"
```

Comment:
NIT: import `each` from `internal` in the above import, as to guarantee there are no circular module deps

Reply:
thanks!, fixed it.